bigchaindb/upsert_validator/validator_election.py

```diff
@@ -0,0 +1,140 @@
+from bigchaindb.common.exceptions import (InvalidSignature,
+                                          DuplicateTransaction)
+from bigchaindb.tendermint.utils import key_from_base64
+from bigchaindb.common.crypto import (public_key_from_ed25519_key)
+from bigchaindb.common.transaction import Transaction, Input, Output
+from bigchaindb.common.schema import (_load_schema,
+                                      _validate_schema,
+                                      TX_SCHEMA_VERSION,
+                                      TX_SCHEMA_COMMON,
+                                      TX_SCHEMA_CREATE)
+
+
+_, TX_SCHEMA_VALIDATOR_ELECTION = _load_schema('transaction_validator_election_' +
+                                               TX_SCHEMA_VERSION, __file__)
+
+
+class ValidatorElection(Transaction):
+
+    VALIDATOR_ELECTION = 'VALIDATOR_ELECTION'
+    # NOTE: this transaction class extends create so the operation inheritence is achieved
```

Comment:
May be it is cleaner to override `cls.create` to raise an exception, what do you think? Or do you want to use `ValidatorElection` for creating election votes later?

Reply:
> May be it is cleaner to override cls.create to raise an exception, what do you think? 
Cloud you please elaborate? What would be achieved if we override `cls.create` and raise an exception here?

Reply:
Currently, there are three methods for instantiating an election transaction - `ValidatorElection.create`, `ValidatorElection.generate`, and the constructor. It is easier to read and use the code if we only keep one. The problem with `create` though is that it comes from the parent so raising an exception is a way to explain that this method is not for use. Ideally, we should have a base transaction class without `create` and `transfer`.

Reply:
> Ideally, we should have a base transaction class without `create` and `transfer`.

That is the target. In this PR I am trying to consolidate the methods which are required in the base `Transaction` class so what other new transaction classes can be written by extending it.